usaspending_api/api_contracts/contracts/v2/search/spending_by_geography.md

```diff
@@ -21,6 +21,12 @@ This endpoint takes award filters, and returns aggregated obligation amounts in
         + `filters` (required, AdvancedFilterObject)
         + `subawards` (optional, boolean)
             True when you want to group by Subawards instead of Awards. Defaulted to False.
+        + `spending_level` (required, enum[string])
```

Comment:
`spending_level` should be `optional` since it can be excluded from the API request and would default to `transactions` in that case.